test/e2e/chaos/basic.go

```diff
@@ -1640,151 +1637,224 @@ selector:
-								Jitter:      "0ms",
-							},
-							Loss: &v1alpha1.LossSpec{
-								Loss:        "25",
-								Correlation: "25",
-							},
-							Duplicate: &v1alpha1.DuplicateSpec{
-								Duplicate:   "25",
-								Correlation: "25",
-							},
-							Corrupt: &v1alpha1.CorruptSpec{
-								Corrupt:     "25",
-								Correlation: "25",
-							},
+				cancel()
+			})
+		})
+
+		ginkgo.Context("[NetworkPartitionWithNetem]", func() {
+			ginkgo.It("[MixSchedule]", func() {
+				ctx, cancel := context.WithCancel(context.Background())
+
+				for index := range networkPeers {
+					err = waitE2EHelperReady(c, ports[index])
+
+					framework.ExpectNoError(err, "wait e2e helper ready error")
+				}
+
+				var (
+					noExcepts           = []*exceptMap{}
+					exceptPartitionPair = []*exceptMap{
+						{
+							Source: "network-peer-2",
+							Target: "network-peer-0",
 						},
-						Duration: pointer.StringPtr("9m"),
-						Scheduler: &v1alpha1.SchedulerSpec{
-							Cron: "@every 10m",
+					}
+				)
+
+				framework.ExpectEqual(len(allBlockedConnection(c, networkPeers, ports)), 0)
+				framework.ExpectEqual(len(allSlowConnection(c, networkPeers, ports, noExcepts)), 0)
+
+				var (
+					testDelayTcParam = v1alpha1.TcParameter{
+						Delay: &v1alpha1.DelaySpec{
+							Latency:     "200ms",
+							Correlation: "25",
+							Jitter:      "0ms",
 						},
-					},
-				}
-				klog.Infof("Injecting delay for 0")
-				err = cli.Create(ctx, complicateNetem.DeepCopy())
+					}
+					testDelayDuration      = pointer.StringPtr("9m")
+					testDelaySchedulerSpec = &v1alpha1.SchedulerSpec{
+						Cron: "@every 10m",
+					}
+				)
+
+				networkPartition := makeNetworkPartitionChaos(
+					ns, "network-chaos-1",
+					map[string]string{"app": "network-peer-2"},
+					map[string]string{"app": "network-peer-0"},
+					v1alpha1.OnePodMode,
+					v1alpha1.OnePodMode,
+					v1alpha1.To,
+					testDelayDuration,
+					testDelaySchedulerSpec,
+				)
+
+				klog.Infof("Injecting partition for 0 first")
+
+				err = cli.Create(ctx, networkPartition.DeepCopy())
 				framework.ExpectNoError(err, "create network chaos error")
 				time.Sleep(5 * time.Second)
-				framework.ExpectEqual(allSlowConnection(), [][]int{{0, 1}, {0, 2}, {0, 3}})
+				framework.ExpectEqual(allBlockedConnection(c, networkPeers, ports), [][]int{{2, 0}})
+
+				networkDelay := makeNetworkDelayChaos(
+					ns, "network-chaos-2",
+					map[string]string{"app": "network-peer-2"},
+					map[string]string{"app": "network-peer-1"},
+					v1alpha1.OnePodMode,
+					v1alpha1.OnePodMode,
+					testDelayTcParam,
+					testDelayDuration,
+					testDelaySchedulerSpec,
+				)
+
+				klog.Infof("Injecting delay for 2 for mix chaos")
+				err = cli.Create(ctx, networkDelay.DeepCopy())
+				framework.ExpectNoError(err, "create network chaos error")
+				time.Sleep(5 * time.Second)
+				framework.ExpectEqual(allSlowConnection(c, networkPeers, ports, exceptPartitionPair), [][]int{{0, 2}, {2, 1}})
```

Comment:
It seems that mix chaos would make peer-0 couldn't visit peer-2, and our spec is:

```
					ns, "network-chaos-1",
					map[string]string{"app": "network-peer-2"},
					map[string]string{"app": "network-peer-0"},
					v1alpha1.OnePodMode,
					v1alpha1.OnePodMode,
					v1alpha1.To
```

it's direction is `network-peer-2` **To** `network-peer-0`. The expected result should be `peer-2` could NOT visit `peer-0`, but `peer-0` could visit `peer-2`, right? @YangKeao